docs/adasum_user_guide.rst

```diff
@@ -0,0 +1,214 @@
+.. inclusion-marker-start-do-not-remove
+
+AdaSum with Horovod
+=================
+
+The Adaptive Summation, or AdaSum is a novel algorithm for improving distributed data parallel training of Deep Learning models. This improvement can be seen in different ways- reducing the number steps to achieve the same accuracy in some cases and allowing to scale to more training workers without penalizing learning rate and convergence stability.
```

Comment:
- Add a comma after "or AdaSum"
- Replace "ways-" with "ways: "
- "allowing to --> allowing you to"